ui-tests/src/test/resources/features/integrations/gmail.feature

```diff
@@ -0,0 +1,112 @@
+    And click on the "Done" button
+
+    # add data mapper step
+    When click on the "Add a Step" button
+    And select "Data Mapper" integration step
+    Then check visibility of data mapper ui
+    Then create data mapper mappings
+      | company | text |
+    And scroll "top" "right"
+    And click on the "Done" button
+
+    # finish and save integration
+    When click on the "Save as Draft" button
+    And set integration name "Integration_gmail_send"
+    And click on the "Publish" button
+
+    # assert integration is present in list
+    Then check visibility of "Integration_gmail_send" integration details
+    And navigate to the "Integrations" page
+
+    And Integration "Integration_gmail_send" is present in integrations list
+    # wait for integration to get in active state
+    Then wait until integration "Integration_gmail_send" gets into "Published" state
+
+    #give gmail time to receive mail
+    Then sleep for "10000" ms
+    Then check that email from "jbossqa.fuse@gmail.com" with subject "syndesis-test" and text "Red Hat" exists
+    And delete emails from "jbossqa.fuse@gmail.com" with subject "syndesis-test"
+
+#
+#  2. Receive an e-mail
+#
+  @gmail-receive
+  Scenario: Receive an e-mail
+
+    # create integration
+    And click on the "Create Integration" button to create a new integration.
+    Then check visibility of visual integration editor
+    And check that position of connection to fill is "Start"
+
+    When select the "My GMail Connector" connection
+    And select "Receive Email" integration action
+    And fill in values
+      | labels | from:jbossqa.fuse@gmail.com |
+    And click on the "Done" button
+
+    Then check that position of connection to fill is "Finish"
+
+    When select the "PostgresDB" connection
+    And select "Invoke SQL" integration action
+    Then fill in invoke query input with "insert into CONTACT values ('Red Hat' , 'Prokop' , 'Dvere' , 'some lead', '1999-01-01')" value
+    And click on the "Done" button
+
+    # finish and save integration
+    When click on the "Save as Draft" button
+    And set integration name "Integration_gmail_receive"
+    And click on the "Publish" button
+
+    # assert integration is present in list
+    Then check visibility of "Integration_gmail_receive" integration details
+    And navigate to the "Integrations" page
+    And Integration "Integration_gmail_receive" is present in integrations list
+    # wait for integration to get in active state
+    Then wait until integration "Integration_gmail_receive" gets into "Published" state
+
+    #give gmail time to receive mail
+    Then send an e-mail
+    #there is 30s pull time in gmail and delay when an e-mail is sent so we have to wait here
+    Then sleep for "60000" ms
+    Then check that query "select * from contact where name = 'Prokop' AND surname = 'Dvere'" has some output
+    And delete emails from "jbossqa.fuse@gmail.com" with subject "syndesis-test"
+
+
+
```

Comment:
remove unnecessary empty lines

\+ if you want to use "And", do it properly for the whole file :-)